storage/innobase/dict/dict0dict.cc

```diff
@@ -914,6 +914,43 @@ dict_table_autoinc_unlock(
 {
 	mutex_exit(table->autoinc_mutex);
 }
+
+/********************************************************************//**
+Create and initialize the analyze index lock for a given table.
+@param[in,out]	table_void	table whose autoinc latch to create
+*/
+static
+void
+dict_table_analyze_index_alloc(
+	void*	table_void)
```

Comment:
do we need void* here?

Reply:
do_or_wait_for_done passes arguments via void*, so yes - we need void* here and then cast to expected parameter type.